src/graphql/mutations/create_game_piece_actions.ts

```diff
@@ -79,7 +80,7 @@ async function handleMoveAction(
   gamePhase: Models.GamePhase,
   gamePiece: Models.GamePiece,
   moveInput: Types.GamePieceMoveActionInput,
-  transaction
+  transaction?: Transaction
```

Comment:
Made this change all over - previously using the implicit `any` type, which is bad TS practice.  This change means "you don't have to pass a transaction, but if you do it must be a `Transaction`